docs/features/cookie-categories.mdx

```diff
@@ -10,9 +10,14 @@ Cookie Categories are used to organize your cookies into groups. You can create
 - Essential
 - Analytics
 - Marketing
+- Personalization
 
 Users will be consenting to these groups of cookies. Each group will have a toggle switch to allow users to opt-in or opt-out of the category.
 
-## Cookie Details
+## Services
 
-Cookie Details are used to describe the cookies used on your website. The required fields are the cookie name and category. The other fields are optional but recommended. You can use CookieChimp's [AI Auto-fill](/docs/ai) feature to automatically fill in the cookie details.
```

Comment:
Do we still want to keep something around cookie details saying how they can use AI etc?

Reply:
Seems a bit out of place here, there's already the section in the AI doc